src/settings/components/Capabilities/CapabilitiesSection.js

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { noop } from 'lodash';
 
 import { FormattedMessage } from 'react-intl';
 
 import { CapabilitiesSettings } from './CapabilitiesSettings';
 import { CapabilitiesProcedural } from './CapabilitiesProcedural';
 import { CapabilitiesDataType } from './CapabilitiesDataType';
 
-const CapabilitiesSection = ({ capabilities, readOnly, onChangeCapabilityCheckbox }) => (
+const CapabilitiesSection = ({ capabilities, readOnly, onChangeCapabilityCheckbox, defineIsCapabilitySelected = noop }) => (
```

Comment:
Other components do not include a default value for this prop. What is special about this one?

Reply:
Fixed, it was temporary solution